js/tests/unit/tooltip.js

```diff
@@ -563,7 +563,6 @@ $(function () {
       $tooltip.bootstrapTooltip('show')
     } catch (err) {
       passed = false
-      console.log(err)
```

Comment:
Maybe this is useful to keep?

Reply:
don't think so because, we set `passed` to false, so our unit test will fail